src/main/java/uk/gov/hmcts/reform/adoption/adoptioncase/model/CaseData.java

```diff
@@ -185,6 +188,14 @@
     )
     private String applicant2SotFullName;
 
+    @CCD(
+        label = "Documents generated",
+        typeOverride = Collection,
+        typeParameterOverride = "AdoptionDocument",
+        access = {DefaultAccess.class}
+    )
+    private List<ListValue<AdoptionDocument>> documentsGenerated;
+
```

Comment:
change DefaultAccess.class to CollectionAccess.class

Reply:
Resolved